src/Membership/MembershipRequest.php

```diff
@@ -12,39 +12,40 @@
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
- * @AssertUniqueMembership(groups={"Default", "Registration"})
+ * @AssertUniqueMembership(groups={"Registration", "Update"})
```

Comment:
Do we really need `Update` here? Seems we cannot edit the email.

Reply:
No thanks

Reply:
In fact yes because we can edit the email when editing user profile.

Reply:
I see 👌